src/esbmc/options.cpp

```diff
@@ -92,6 +92,9 @@ const struct group_opt_templ all_cmd_options[] = {
     {"show-loops", NULL, "show the loops in the program"},
     {"show-claims", NULL, "only show claims"},
     {"show-vcc", NULL, "show the verification conditions"},
+    {"show-caller-loc",
+     NULL,
+     "show the location of the function calling expression"},
```

Comment:
Should we rename this to a more universally known `show-stacktrace` and mention the counter-example in the description of the option?

Reply:
Changed as suggested.